src/Concerns/HasTenantOptions.php

```diff
@@ -23,6 +27,9 @@ protected function getTenants(): LazyCollection
             ->when($this->option('tenants'), function ($query) {
                 $query->whereIn(tenancy()->model()->getTenantKeyName(), $this->option('tenants'));
             })
+            ->when(! is_null($this->option('with-pending')) && tenancy()->model()::hasGlobalScope(PendingScope::class), function ($query) {
```

Comment:
Instead of checking for null, could we change the option above in `getOptions()` to be a boolean? Default to `false` and if specified, it will be true.

Reply:
Right, that would make more sense

Reply:
But maybe not – `with-pending` is a thing you'd want to specify because of the `pending.include_in_queries` config option. When that is `true`, you don't have to pass `--with-pending` to the command to include the pending tenants in it. In that case, defaulting to false would be wrong

Reply:
Can we grab the default from the `pending.include_in_queries ` config?

Reply:
I see that the option is `InputOption::VALUE_NONE`, which means it was meant to be passed only as `--with-pending` (without `=true` or `=false`). Making it accept `true`/`false` and default to the config value seems better though.

Reply:
The option works weirdly – not passing `--with-pending` defaults to the config value (as it should), but just `--with-pending` is recognized as `null` when it should be `true`. Passing `true`/`false` (or anything else) is recognized as a string, so we'll have to do `(bool) $this->option('with-pending')`